### DISCUSSION TITLE
Renovate opens PR, but only changes yarn.lock file, not package.json

### DISCUSSION THREAD

**Discussion by tHeMaskedMan981** [renovatebot/renovate#28455] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

I added renovate bot recently to a repo.  It worked fine initially, but recently it is generating PRs with only changes to the yarn.lock file, not the package.json file.  I don't know if I messed up something in the configuration, or maybe it doesn't update for patch versions, but I am not able to get proper PRs. repo is private so can't share PR, so adding screenshots . 
<img width="1347" alt="Screenshot 2024-04-17 at 1 24 16 AM" src="https://github.com/renovatebot/renovate/assets/24896672/96f66ca6-e859-48b2-aa9c-c5024d8d5905">
<img width="1882" alt="Screenshot 2024-04-17 at 1 24 30 AM" src="https://github.com/renovatebot/renovate/assets/24896672/23938112-f913-4b59-b543-adc3eda71706">

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
This is expected/intended behavior if you are using ranges in your `package.json`. Read the docs for `rangeStrategy` if you wish to change this
